test/nginx_default_ingress_ctl_test.go

```diff
@@ -0,0 +1,84 @@
+package integration_tests
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/retry"
+	. "github.com/onsi/gomega"
+
+	"github.com/ministryofjustice/cloud-platform-infrastructure/test/helpers"
+)
+
+var _ = Describe("Nginx Default Ingress v1", func() {
+	var (
+		currentCluster = c.ClusterName
+		namespaceName  = fmt.Sprintf("smoketest-ingress-v1-%s", strings.ToLower(random.UniqueId()))
+		host           = fmt.Sprintf("%s-nginx.apps.%s.%s", namespaceName, currentCluster, domain)
+		options        = k8s.NewKubectlOptions("", "", namespaceName)
+		url            = fmt.Sprintf("https://%s", host)
+		class          = "nginx"
+		tpl            string
+	)
+
+	BeforeEach(func() {
+		By("not having an ingress resource deployed")
+
+		Expect(helpers.HttpStatusCode(url)).To(Equal(404))
+
+		k8s.CreateNamespace(GinkgoT(), options, namespaceName)
+	})
+
+	AfterEach(func() {
+		defer k8s.KubectlDeleteFromString(GinkgoT(), options, tpl)
+		defer k8s.DeleteNamespace(GinkgoT(), options, namespaceName)
+	})
+
+	Context("when ingress resource is deployed using 'nginx' ingress controller", func() {
```

Comment:
can u change the context to say "when ingress resource is deployed using 'default' ingress controller"